### ISSUE TITLE
Attack with more troops near collectors

### ISSUE THREAD

**Issue by mk89pwnz** [cool7su/Broken_Clashbot#38]
I noticed some other bots have this feature. Is this possible to implement in this version of bot? or a lot of job to make it happen? btw I saw gamebot will have this feature in the next release :)

**Comment by codebroken** [comment]
Really like this idea. Shouldn't be too difficult, images already exist to find elixir collectors just need to add in gold mines and DE drills. Attack algorithm logic is the real killer but I'm confident I can do it but it'll take a back seat to some of the more easy stuff.

**Comment by shariqriaz** [comment]
u should join gamebot.org and merge your bot with them..its a very good open source bot forum and the version is much more stable ..its missing only DE zap and dual attack i.e. DEAD+LIVE BASE ..they always appreciate the contribution :)

**Comment by fmonera** [comment]
I agree. This effort should be merged with Gamebot.org. A lot of developers there. Very stable code.

**Comment by shariqriaz** [comment]
yup so much appreciation and truly an open source and free :)
